### PR TITLE
fix up formatting issues

### PR DESCRIPTION
**Pull request by nslater**
minor fixes

### CODE REVIEW

File: DEVELOP.rst [hunk 1]
```diff
@@ -44,17 +44,17 @@ Distributing
...
- - Push to origin on the master branch
- - Run ``./devtools/create_tag.sh``
- - Run ``bin/py setup.py sdist upload``
+- Update the version in ``src/crate/theme/rtd/__init__.py``
+- Update the version in ``package.json``
+- Add section to the ``CHANGES.txt`` file
+- Commit your changes with a message like "prepare release x.x.x"
+- Push to origin on the master branch
+- Run ``./devtools/create_tag.sh``
+- Run ``bin/py setup.py sdist upload``
```
Comment (mfussenegger) [inline DEVELOP.rst]: It's recommended to use twine instead of setup.py upload -> https://packaging.python.org/distributing/#uploading-your-project-to-pypi

Comment (nslater) [inline DEVELOP.rst]: @mfussenegger am I okay to merge as is for the formatting fixes and then do another PR for updating the docs? or should I combine changes?

Comment (mfussenegger) [inline DEVELOP.rst]: Sounds good to me
